### DISCUSSION TITLE
Flatpak issue with org.gnome.platform

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#426] [Q&A] [answered]
Hi there. Trying to update my flatpaks and getting this error:

```
Info: runtime org.gnome.Platform branch 44 is end-of-life, with reason:
   The GNOME 44 runtime is no longer supported as of March 20, 2024. Please ask your application developer to migrate to a supported platform.
Info: applications using this runtime:
   info.portfolio_performance.PortfolioPerformance
```

flatpak list

![grafik](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/b917fd6a-b1b4-40eb-a584-5d59d1b5ab42)

Is this something that can be fixed? by Distro, Debian, etc...?

Thanks

**Comment by geckolinux** [comment]
Hi there, the Flatpak infrastructure is managed by Flathub.org. You could probably remove the old runtime:
`flatpak remove org.gnome.Platform` and select the number corresponding to branch 44.

**Comment by AlphaElwedritsch** [comment]
ok, i tried this but i cannot remoce v44. portfolioperformance depends on it
when i try to replace v44 with a newer one, system will deinstall portfolioperformance also

think have to contact the developer, if he can switch to newer version

**Comment by FabioLolix** [comment]
Hello, this have already been reported to the flathub repo https://github.com/flathub/info.portfolio_performance.PortfolioPerformance/issues/106

btw the flatpak is only repackaging the upstream binary

[ "url": "https://github.com/portfolio-performance/portfolio/releases/download/0.68.3/PortfolioPerformance-0.68.3-linux.gtk.x86_64.tar.gz",](https://github.com/flathub/info.portfolio_performance.PortfolioPerformance/blob/5207fef671f65307ed9e965640d8cd976cd5beb3/info.portfolio_performance.PortfolioPerformance.json#L58)

**Comment by AlphaElwedritsch** [comment]
OK, it's addressed and should be fixed in the comming days :)

**Comment by AlphaElwedritsch** [comment, marked as answer]
it is fixed
0.68.4 base on org.gnome.Platform 46
